src/cast.c

```diff
@@ -1386,6 +1386,31 @@ Type *toStaticArrayType(SliceExp *e)
 }
 
 /* ==================== castTo ====================== */
+/**
+ * This function should be called by iterateAliasThis.
```

Comment:
This function is written to be a delegate to `iterateAliasThis`.
C++ doesn't allow us to write function inside another function, thus we can't write delegate declaration "in place".